src/outlineView.ts

```diff
@@ -88,7 +90,12 @@ export class OutlineView {
   }
 
   // callback for scrolling and highlighting the element that the cursor is on
-  selectAtCursorLine(newBufferPosition: CursorPositionChangedEvent["newBufferPosition"]) {
+  selectAtCursorLine(cursor: Cursor | undefined) {
+    // no cursor
+    if (!cursor) {
```

Comment:
Honestly, it's _really_ hard to get an editor into a state where there are no cursors, and it's thoroughly borked at that point anyway. Feel free to leave this check in of course, but it's somewhat redundant.

Reply:
If the editor gets out of the focus, the cursor gets hidden. I am not sure why this is not covered in the types.

Reply:
Hidden -- yes. Removed -- no.

Reply:
Oh, I see